.github/workflows/linting.yml

```
@@ -124,6 +132,11 @@ jobs:
               - '**/eslint.config.*'
               # If the excludelist changed, run to ensure newly non-excluded files pass.
               - 'tools/eslint-excludelist.json'
+            misc_css:
+              # If Stylelint config changed, there may be new checks.
+              - .stylelintignore
+              - stylelint.config.mjs
```

Is it possible to have a stylelint config in a subdir, like eslint does? If so, adding something like this might be a good idea.
```suggestion
              - stylelint.config.mjs
              - '**/stylelint.config.{js,mjs,cjs}'
```

---

It only accepts a single config file, and [searches from the current working dir upward](https://stylelint.io/user-guide/configure/), so effectively only the one the current working dir is relevant.

---

Remember it was very unclear whether the "searches from the current working directory" actually meant that or if it searches from the _current file's_ directory until it finds a config file.

---

Ah, indeed. In fact I tested and it does seem to be relative to the current file, so I'll use your suggestion.